packages/api/src/controllers/stream.ts

```diff
@@ -839,14 +840,23 @@ app.get("/:id", authorizer({}), async (req, res) => {
   res.json(db.stream.addDefaultFields(stream));
 });
 
-// returns stream by steamKey
+// returns stream by playbackId
 app.get("/playback/:playbackId", authorizer({}), async (req, res) => {
```

Comment:
I'm a little lost. I thought that the issue was that we were flooding with `/lockPull` requests, so why to add the cache to `/playback/<playbackId>`? 

It the DB peak was related to the `GET /playback/<playbackId>`?

Reply:
The flood I am seeing is to that endpoint, caused by catalyst-api using the method `GetStreamByPlaybackId` from go-api-client

Reply:
Ok 👌